devdox/app/services/repository.py

```
@@ -208,13 +222,18 @@
             
             return str(saved_repo.id)
 
-        except IntegrityError:
-            raise BadRequest(reason=REPOSITORY_ALREADY_EXISTS)
+        except DevDoxModelsException as e:
+            if e.error_type == RepoErrors.REPOSITORY_ALREADY_EXIST.value["error_type"]:
+                raise BadRequest(reason=REPOSITORY_ALREADY_EXISTS) from e
+            raise
+        except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)